### PR TITLE
Deprecate Text.Sexp

### PR DESCRIPTION
**Pull request by ujihisa**
See also: https://github.com/vim-jp/vital.vim/pull/587#issuecomment-396168465

### CODE REVIEW

File: doc/vital.txt [hunk 1]
```diff
@@ -209,7 +209,7 @@ LINKS					*Vital-links*
   |vital/Text/LTSV.txt|		LTSV library.
   |vital/Text/Lexer.txt|	lexer library.
   |vital/Text/Parser.txt|	parser library.
-  |vital/Text/Sexp.txt|		S-Expression parser.
+  |vital/Deprecated.Text.Sexp.txt|		S-Expression parser.
```
Comment (tyru) [inline doc/vital.txt]: dot should be slash.

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Module						 | Description
 [Text.LTSV](doc/vital/Text/LTSV.txt)		 | LTSV library
 [Text.Lexer](doc/vital/Text/Lexer.txt)		 | lexer library
 [Text.Parser](doc/vital/Text/Parser.txt)	 | parser library
-[Text.Sexp](doc/vital/Text/Sexp.txt)	         | S-Expression parser
+[Deprecated.Text.Sexp](doc/vital/Deprecated.Text.Sexp.txt)	         | S-Expression parser
```
Comment (tyru) [inline README.md]: dot should be slash.

File: doc/vital/Deprecated/Text/Sexp.txt [hunk 3]
```diff
@@ -1,27 +1,27 @@
-*vital/Text/Sexp.txt*		S-Expression parser.
+*vital/Deprecated.Text.Sexp.txt*		S-Expression parser.
```
Comment (tyru) [inline doc/vital/Deprecated/Text/Sexp.txt]: dot should be slash.

File: doc/vital.txt (hunk 1 above)
Comment (ujihisa) [reply doc/vital.txt]: ウギャー一斉置換したときにがばっとミスってました。あぶなすぎる

Comment (ujihisa) [reply doc/vital.txt]: fixed all 3 at 1b117a9dd0d5d9d3dff31dcf6187b1332e486dc5

File: README.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ Module						 | Description
 [Text.LTSV](doc/vital/Text/LTSV.txt)		 | LTSV library
 [Text.Lexer](doc/vital/Text/Lexer.txt)		 | lexer library
 [Text.Parser](doc/vital/Text/Parser.txt)	 | parser library
-[Text.Sexp](doc/vital/Text/Sexp.txt)	         | S-Expression parser
+[Deprecated.Text.Sexp](doc/vital/Deprecated/Text/Sexp.txt)	         | S-Expression parser
```
Comment (thinca) [inline README.md]: Deprecated 名前空間以下のモジュールは現状 README や doc/vital.txt のモジュール一覧に記載されていないので、合わせた方が良いかと。

Comment (ujihisa) [reply README.md]: True. Fixed at 949f6edc0176fa8951898870af3a93f073ff401e. thx!

Comment (thinca) [reply README.md]: help も頼むー。

Comment (ujihisa) [reply README.md]: Oops, why didn't I notice that? Thanks @thinca, good catch! fixed at 33847a7cf761282d14e2f5ba997699ab4f9f2cf1

File: doc/vital/Deprecated/Text/Sexp.txt [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+INTERFACE			|Vital.Deprecated.Text.Sexp-interface|
+  FUNCTIONS			  |Vital.Deprecated.Text.Sexp-functions|
 
 ==============================================================================
-INTRODUCTION				*Vital.Text.Sexp-introduction*
+INTRODUCTION				*Vital.Deprecated.Text.Sexp-introduction*
 
-*Vital.Text.Sexp* is a S-Expression parsing library. For performance it uses
+*Vital.Deprecated.Text.Sexp* is a S-Expression parsing library. For performance it uses
 lua, so the Vim that runs this needs |+lua|.
```
Comment (rhysd) [inline doc/vital/Deprecated/Text/Sexp.txt]: How about adding a notation that this module was deprecated? (though it's obvious from its module name...)

Comment (ujihisa) [reply doc/vital/Deprecated/Text/Sexp.txt]: That's also an option, but I feel that it's verbose to repeat there. (Alternative option would be to insert warning message when you use, but before that the plugin author need to rename the module, so it's still verbose)

**Comment by ujihisa** [comment]
Thanks @tyru @thinca @rhysd !

**Pull request merged**